common/production_methods/07_government.txt

```diff
@@ -523,6 +523,59 @@ pm_urban_academia = {
 			country_prestige_add = 1
 		}
 	}
+	
+}
+pm_placeholder_1 = {
+	texture = "gfx/interface/icons/production_method_icons/government_run.dds"
+	building_modifiers = {
+		
+		level_scaled = {
+			building_employment_academics_add = 25
+            building_employment_clerks_add = 50
+            building_employment_laborers_add = 75
+		}
+		workforce_scaled = {
+			goods_input_paper_add = 0.20
+			goods_input_services_add = 0.20
+		}	
+	}
+	state_modifiers = {
+		workforce_scaled = {
+			state_education_access_add = 0.001
+		}
```

Comment:
I thought we agreed that this would be level_scaled?

Reply:
Still not answered

Reply:
Yep, just didn't write it down anywhere